src/Vimeo/Vimeo.php

```diff
@@ -73,7 +73,7 @@ public function __construct($client_id, $client_secret, $access_token = null)
      * @param bool $json_body
      * @return array This array contains three keys, 'status' is the status code, 'body' is an object representation of the json response body, and headers are an associated array of response headers
      */
-    public function request($url, $params = array(), $method = 'GET', $json_body = true)
+    public function request($url, $params = array(), $method = 'GET', $json_body = true, $last_modified = null)
```

Comment:
I think this would be better in the long run as a $headers array parameter, that could include the If-Modified-Since header